tests/components/mqtt/test_init.py

```diff
@@ -1172,7 +1172,7 @@ async def test_publish_error(hass, caplog):
 
 
 async def test_handle_message_callback(hass, caplog, mqtt_mock, mqtt_client_mock):
-    """Test for handling an incoming message callback."""
+    """Test for handling an incoming message callback via _mqtt_on_message."""
```

Comment:
We shouldn't include the name of the function in the docstring. It's a detail and not important. We may change the name of the integration callback in the future. This test will still work then.

Reply:
Ok